### ISSUE TITLE
Android error

### ISSUE THREAD

**Issue by lawlietop123** [AifeiI/qr_code_tools#14]
I found a problem in version 0.1.0
![截圖 2024-05-20 下午3 24 47](https://github.com/AifeiI/qr_code_tools/assets/37841735/31f36488-6937-4d64-8dae-577f353f55ab)

andoird gradle plugin version error

**Comment by zhangyc** [comment]
> I found a problem in version 0.1.0 ![截圖 2024-05-20 下午3 24 47](https://private-user-images.githubusercontent.com/37841735/331979347-31f36488-6937-4d64-8dae-577f353f55ab.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg3MDI4OTksIm5iZiI6MTcxODcwMjU5OSwicGF0aCI6Ii8zNzg0MTczNS8zMzE5NzkzNDctMzFmMzY0ODgtNjkzNy00ZDY0LThkYWUtNTc3ZjM1M2Y1NWFiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE4VDA5MjMxOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBhN2RiNGMyNDEwYmEwMDk2ZmVjNDdhMGM5YWI4YzNjNWNiMjEyZTgwNzE1MTcwZDBhNTMwNzIzMTkxY2E1YTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.mFw2w63Lz-N2oUqM_ksWXoE76O2j7pYiE_oi0Ce_TAI)
> 
> andoird gradle plugin version error

![image](https://github.com/AifeiI/qr_code_tools/assets/10071342/30a0e766-80c4-4114-8d57-5c155c8e9550)

**Comment by dipu-polygontech** [comment]
* What went wrong:
The Android Gradle plugin supports only Kotlin Gradle plugin version 1.5.20 and higher.
The following dependencies do not satisfy the required version:
project ':qr_code_tools' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.31


same error

**Comment by VijaykumarTR005** [comment]
Hi @dipu-polygontech , even i am also facing the same issue if you find the solution share it here.

**Comment by dipu-polygontech** [comment]
> Hi @dipu-polygontech , even i am also facing the same issue if you find the solution share it here.

nah, i have change packages, Now using mobile_scanner 5.1.1, analyzeImage function.

`
 MobileScannerController _controller = MobileScannerController();

  Future<String?> _scanQrCodeFromImage(File image) async {
    final barcodeFinder = await _controller.analyzeImage(image.path);

    if (barcodeFinder != null && barcodeFinder.barcodes.isNotEmpty) {
      return barcodeFinder.barcodes.first.rawValue;
    }

    return null;
  }`

**Comment by macdipu** [comment]
> Hi @dipu-polygontech , even i am also facing the same issue if you find the solution share it here.

nah, i have change packages, Now using mobile_scanner 5.1.1, analyzeImage function.

```
 MobileScannerController _controller = MobileScannerController();

  Future<String?> _scanQrCodeFromImage(File image) async {
    final barcodeFinder = await _controller.analyzeImage(image.path);

    if (barcodeFinder != null && barcodeFinder.barcodes.isNotEmpty) {
      return barcodeFinder.barcodes.first.rawValue;
    }

    return null;
  }
```

**Comment by MuhammadUsamaSiddiqui** [comment]
Facing the same issue.
